generators/addons/index.js

```diff
@@ -151,18 +160,18 @@ module.exports = class extends Generator {
 
         // azure deevops configuration
         let adoFileName = 'azure-pipelines.yml';
-        // Karma configuration
-        let karmaFileDestPath = 'config/karma.config.js';
-        let karmaFileSourcePath = 'karma.config.js';
+        // Jest configuration
+        let jestFileDestPath = 'config/jest.config.json';
```

Comment:
@StfBauer this should be done at the jest selection step, I probably need to remove it right?

Reply:
@baywet Yes, I think. We need to test it but this will be done via the post install.